### DISCUSSION TITLE
Change background

### DISCUSSION THREAD

**Discussion by SFXToto** [Eight-P/BetterDiscord-Stuff#28] [Q&A] [answered]
Hello, I would like to know if it is possible to change the background,
Because the basis of the theme, I like it very much,
But I don't like the gray background at all,
Would I like to make it black or something?
Is it possible to know how to do this?
THANKS

**Comment by Eight-P** [comment, marked as answer]
Yes, you can change the background image or make it a solid color, although the same color would then be applied to the whole app just like the image is, so you could not have the member list in a different color than the chat for example (unless you add some code yourself).

Check [this](https://github.com/Eight-P/BD.8P/tree/master/Themes) quick explanation on how to use variables.
Then change the `--background-app` and `--background-server` variables.

- You can change the links to any HTTPS image link e.g.:
  `--background-app: url("https://someurl");`

- You can make them a solid color e.g.:

  `--background-app: black ;` or `--background-app: rgb(3, 18, 46);`

- Or a gradient e.g.:

  `--background-app: linear-gradient(180deg,rgb(6, 6, 83) 0%, rgba(0,255,201,1) 100%);`

- You can also remove the server background with:

  `--background-server: transparent ;`

#### Make sure you save your changes to make them apply.
